### PR TITLE
doc: Update database doc to match uPortal 5.

### LINKED ISSUE #141: Update database documentation to match current version

### PR DESCRIPTION
**Pull request by bjagg**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] documentation is changed or added

##### Description of change

 Fixes #141.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: docs/database/index.md [hunk 1]
```diff
@@ -6,86 +6,71 @@ uPortal is configured to use a file-based HSQL database by default.
...
+
+```groovy
+    dependencies {
+        /*
+         * Add additional JDBC driver jars to the 'jdbc' configuration below;
+         * do not remove the hsqldb driver jar that is already listed.
+         */
+        jdbc "org.hsqldb:hsqldb:${hsqldbVersion}"
+        jdbc "com.microsoft.sqlserver:mssql-jdbc:${mssqlJdbcVersion}"
+
```
Comment (ChristianMurphy) [inline docs/database/index.md]: Missing a closing tag for `dependencies`?

Comment (bjagg) [reply docs/database/index.md]: It's a snippet ...

Comment (ChristianMurphy) [reply docs/database/index.md]: ahhh, got it, maybe remove the trailing newline?
I misread this as a self contained snippet.

**Comment by drewwills** [comment]
Thanks @bjagg, +1.  👍

**Pull request merged**